dashboards/resources.libsonnet

```
@@ -1 +1,3 @@
-(import 'resources/cluster.libsonnet')
+(import 'resources/cluster.libsonnet') {
+  grafanaDashboards+:: (import 'resources/pod.libsonnet').grafanaDashboards,
+}
```

Does the same syntax as upstream work?

```suggestion
(import 'resources/cluster.libsonnet') +
(import 'resources/pod.libsonnet')
```